components/navbar/navbar.snip.html

```diff
@@ -77,9 +78,17 @@
           {{> ../calltracking/calltracking.snip.html}}
         {{/calltracking}}
       </div>
+      {{#state}}
```

Comment:
`state` adds a wrapper div which allows classes to be changed. This is being used on the product-details page when a product has been added to the cart.